package.json

```diff
@@ -66,9 +66,14 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "testem ci",
-    "lint-dts": "tslint sweetalert2.d.ts --format verbose",
-    "check-dts": "tsc sweetalert2.d.ts && npm run lint-dts",
+    "start": "gulp watch",
+    "fix:lint": "standard --fix",
+    "test": "npm run build && npm run check",
+    "build": "gulp build",
+    "check": "npm run check:bundlesize && npm run check:tests && npm run check:ts",
+    "check:bundlesize": "bundlesize -f dist/sweetalert2.all.min.js -s 15kB",
+    "check:tests": "testem ci",
+    "check:ts": "tsc sweetalert2.d.ts",
```

Comment:
Removed `"lint-dts"` because linting is covered by the build process.

Reply:
:+1: